docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -10,6 +10,7 @@
   * [Resin](http://www.caucho.com/resin-4.0/) 4 (Optional¹)
   * [Jetty Server](http://www.eclipse.org/jetty/) 9
   * [Spring Webflux](https://docs.spring.io/spring/docs/current/spring-framework-reference/web-reactive.html) 5.x
+  * [Spring Cloud Gateway](https://spring.io/projects/spring-cloud-gateway) 2.1.x.RELEASE
```

Comment:
Please add optional tag.

Reply:
ok,  move `Spring Cloud Gateway` at `HTTP Server` to `HTTP Gateway` is better?